src/Ucommerce.Avenue.Umbraco/Views/PartialView/ProductReview.cshtml

```diff
@@ -102,7 +102,7 @@ using (Html.BeginUmbracoForm("Index", "Review"))
             <div class="control-group margin-top">
                 <div class="controls">
                     <button type="submit" class="btn btn-primary" name="review-product">Save</button>
-                    <input type="hidden" name="reviewed-product-sku" id="@product.Sku" />
+                    <input type="hidden" id="reviewed-product-sku" name="ProductSku" value="@product.Sku" />
```

Comment:
Not sure about this... Please make sure it doesn't break anything. Is changing this even necessary?

Reply:
This change is what made it possible to grab the Sku in the controller. I think this has always been a mistake, but it just wasn't used to anything, so it wasn't discovered until now.

Reply:
Good catch!

Reply:
At least half the credit goes to @skovsboll  :p But thanks!